### DISCUSSION TITLE
Multiple triggers for Lottie widget

### DISCUSSION THREAD

**Discussion by oceandiveloper** [elementor/elementor#21173] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

At the moment we can choose only 1 trigger for the animation of a Lottie widget: "Viewport", "On Click", "On Hover" or "Scroll". In my current client project the client would like to have 2 triggers for the same Lottie widget: "Viewport" AND "On Hover". Like this the animation can be triggered once the user scrolls to the Lottie widget (to grab attention) and another time whenever the user hovers over the linked element.

### Describe the solution you'd like

Please add the possibility to add multiple triggers for the same widget.

**Comment by nicholaszein** [comment]
Hello @oceandiveloper,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. 

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.

Best regards
